### PR TITLE
26 docker compose dev env

### PR DESCRIPTION
**Pull request by eddyhakimi**

### CODE REVIEW

File: src/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+services:
+  frontend:
+    container_name: frontend
+    user: "${USER_ID}:${GROUP_ID}" # prevent root files in host system
+    build:
+      context: ./deskstar-frontend
+    ports:
+      - 3000:3000
+    volumes:
+      - ./deskstar-frontend:/app/code
```
Comment (n3rdc4ptn) [inline src/compose.yaml:10]: Maybe we should cache the volume mount for better performance:
https://stackoverflow.com/questions/43844639/how-do-i-add-cached-or-delegated-into-a-docker-compose-yml-volumes-list

File: src/compose.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    container_name: postgres
+    build: 
+      context: ./deskstar-db
+    user: "postgres"
+    environment:
+      - POSTGRES_USER
+      - POSTGRES_DB
+      - POSTGRES_PASSWORD
+    volumes:
+      - ./deskstar-db/init:/docker-entrypoint-initdb.d
```
Comment (n3rdc4ptn) [inline src/compose.yaml:39]: maybe also here
https://stackoverflow.com/questions/43844639/how-do-i-add-cached-or-delegated-into-a-docker-compose-yml-volumes-list

**Review by n3rdc4ptn — CHANGES_REQUESTED** [review]
Looks good to me, I would suggest that you change the volume mount to be a cached volume mount. I experienced better performance with it in past projects.
https://stackoverflow.com/questions/43844639/how-do-i-add-cached-or-delegated-into-a-docker-compose-yml-volumes-list

File: src/compose.yaml (hunk 2 above)
Comment (eddyhakimi) [reply src/compose.yaml]: Good point. I applied your changes.

File: src/compose.yaml (hunk 1 above)
Comment (eddyhakimi) [reply src/compose.yaml]: Good point. I applied your changes.

**Review by n3rdc4ptn — APPROVED** [review]

**Pull request merged**